src/app/search/search.component.ts

```diff
@@ -136,7 +136,7 @@ export class SearchComponent implements OnInit, OnDestroy {
 
   ngOnInit() {
     // Set alert store back to its initial state. Otherwise an error created on another page might appear when search page is loaded.
-    this.alertService.clearEverything();
+    // this.alertService.clearEverything();
```

Comment:
If this is not used, just remove it

Reply:
Ah forgot to delete that. Canceled other build and pushed new commit.